packages/compiler-sfc/src/script/context.ts

```diff
@@ -163,7 +163,10 @@ export function resolveParserPlugins(
     userPlugins = userPlugins.filter(p => p !== 'jsx')
   }
   if (lang === 'ts' || lang === 'tsx') {
-    plugins.push(['typescript', { dts }])
+    plugins.push(['typescript', { dts }], 'explicitResourceManagement')
+    if (!plugins.includes('importAssertions')) {
+      plugins.push(['importAttributes', { deprecatedAssertSyntax: true }])
```

Comment:
Why support the deprecated syntax here?

Reply:
Currently in typescript@5.2 only support `import assert` syntax (`deprecatedAssertSyntax`). In ts@5.3 it support both.

Reply:
I think we should only support the latest version.

Also, the `includes` check should also check `importAttributes`?

Reply:
The latest version now is TS@5.2. But 5.3 will be released on 2023-11-14, which less than one month. Vue 3.4 could be likely later than it.